indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -151,6 +161,8 @@ public IndexTask(
     super(
         // _not_ the version, just something uniqueish
         makeId(id, ingestionSchema),
+        String.format("index_%s", makeDataSource(ingestionSchema)),
```

Comment:
If groupId is not specified, this is usually simply the same as ID by prior convention.

It is possible that a constructor need be added to AbstractFixedIntervalTask to account for this combination of parameters. Can you modify this so that ID and groupID are the same in the absence of an explicit groupId?